Move C# 13 proposals to subfolder (#8430)
dotnet/csharplang: proposals/csharp-13.0/esc-escape-sequence.md

# String/Character escape sequence `\e`

## Summary
An addition of the string/character escape sequence `\e` as a shortcut/short-hand replacement for the character code point `0x1b`, commonly known as the `ESCAPE` (or `ESC`) character.  This character is currently accessible using one of the following escape sequences:
- `\u001b`
- `\U0000001b`
- `\x1b` (not recommended, see the picture attached at the bottom.)

With the implementation of this proposal, the following assertions should be true:
```csharp
char escape_char = '\e';

Assert.IsTrue(escape_char == (char)0x1b, "...");
Assert.IsTrue(escape_char == '\u001b', "...");
Assert.IsTrue(escape_char == '\U0000001b', "...");
Assert.IsTrue(escape_char == '\x1b', "...");
```

## Detailed design

> I propose the language syntax specification to be changed as follows in section [6.4.5.5](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/lexical-structure.md#6455-character-literals):

Reviewer: ```suggestion
The language syntax specification is changed as follows in section [6.4.5.5](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/lexical-structure.md#6455-character-literals):
```
